Source/WebCore/svg/SVGUseElement.cpp

```diff
@@ -618,6 +618,8 @@ void SVGUseElement::updateExternalDocument()
 {
     URL externalDocumentURL;
     Ref<Document> document = this->document();
+    if (document->completeURL(href()).protocolIsData())
+        return;
```

Comment:
I think we should add a FIXME comment here as well. The data url should be treated as an external URL and we should create an external Document for it. We can add something like:

> FIXME: This early exit should be removed once the ASSERT(!url.protocolIsData()) is removed from isExternalURIReference()